src/main/java/seedu/address/logic/parser/CodeConnectParser.java

```diff
@@ -58,6 +61,15 @@ public Command parseCommand(String userInput) throws ParseException {
         case AddCommand.COMMAND_WORD:
             return new AddCommandParser().parse(arguments);
 
+        case AddTeamCommand.COMMAND_WORD:
+            return new AddTeamCommandParser().parse(arguments);
+
+         case AddContactToTeamCommand.COMMAND_WORD:
+            return new AddContactToTeamCommandParser().parse(arguments);
+
+        case DeleteContactFromTeamCommand.COMMAND_WORD:
+            return new DeleteContactFromTeamCommandParser().parse(arguments);
+
```

Comment:
This needs to be moved under `TeamCommandsParser` instead. In fact, `AddTeamCommand` is already there, so you can take a look at it and see how its implemented.